src/test/java/io/kokuwa/maven/helm/junit/MojoExtension.java

```diff
@@ -106,12 +108,20 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
 			// preconfigure
 
 			mojo.setChartDirectory(new File("src/test/resources/simple")); // set some sane defaults for tests
-			mojo.setHelmExecutableDirectory(new File("src/it")); // avoid download helm
+			mojo.setHelmExecutableDirectory(determineHelmExecutableDirectory());
 			mojo.setHelmVersion("3.12.0"); // avoid github api
 
 			return mojo;
 		} catch (ReflectiveOperationException e) {
 			throw new ParameterResolutionException("Failed to setup mockito.", e);
 		}
 	}
+
+	public static File determineHelmExecutableDirectory() {
```

Comment:
A: I think this is more useable if you return it as a `Path`. Any caller that needs a file can call the `toFile` method.

Reply:
so I did it this way to stay consistent with how the rest of the repo returns Files instead of Paths. for example `setHelmExecutableDirectory()` expects a File, as do other set___Directory functions do